runtime/cluster.cu

```diff
@@ -0,0 +1,93 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+
+#if (defined(__CUDA_ARCH__) && (__CUDA_ARCH__ >= 900))
+
+namespace Hopper {
+
+// The optional .relaxed qualifier on barrier.cluster.arrive specifies that
+// there are no memory ordering and visibility guarantees provided for the
+// memory accesses performed prior to barrier.cluster.arrive.
+void cluster_arrive_relaxed() {
```

Comment:
I think we use the camel case for the runtime symbols as well, so:

```suggestion
void clusterArriveRelaxed() {
```

Reply:
Done